app/views/courses/_lti_form.html.erb

```diff
@@ -0,0 +1,25 @@
+<div id="lti">
+  <h3><%= t('lti.lti_configuration') %></h3>
+  <% unless @lti_deployments.nil? || @lti_deployments.empty? %>
+    <% @lti_deployments.each do |deployment| %>
+      <p>
+        <%= t('lti.lti_course_link_html',
+              { lms_host: deployment.lti_client.host,
+                lms_course_id: deployment.lms_course_id,
+                lms_course_name: deployment.lms_course_name }) %>
+      </p>
+      <%= button_to t('delete'),
+                    { action: 'destroy_lti_deployment',
+                      id: @current_course.id,
+                      lti_deployment_id: deployment.id
+                      },
+                    html: { title: t('lti.unlink_courses') },
+                    data: { confirm: t('lti.unlink_confirm') },
+                    method: 'delete',
+                    remote: false,
+                    class: 'inline-button delete' %>
+    <% end %>
+    <% else %>
```

Comment:
The indentation here is incorrect; the `else` should be unindented, and the following line should be indented.